packages/common/src/utils/common.ts

```diff
@@ -92,16 +92,6 @@ export const IS_PREVIEW_MODE = __mode__ === 'preview';
 // Production mode
 export const IS_PROD_MODE = __mode__ === 'prod';
 
-export function logDev(message: unknown, ...optionalParams: unknown[]) {
```

Comment:
TBH I think it doesn't make a difference because the `logDev("This stupid ..")` call would still be present in the compiled code. I think the compiler only removes `if(false)` blocks which is just the body of the `logDev` function.

Reply:
I think I did test it by searching the production build for the strings and at least at one point they weren't showing up. That might have been several refactors ago though. Not the most important thing, but I thought it was worth mentioning.

Reply:
Hmm, you might be right...